ledger/db/src/ledger_db.rs

```diff
@@ -2400,6 +2401,7 @@ mod ledger_db_test {
                 &last_block,
                 &Default::default(),
                 &block_contents,
+                last_block.timestamp + 1,
```

Comment:
Where possible increase previous timestamp by 1 to conform with logic coming down the line to ensure timestamps are valid.